packages/as-proto-gen/src/generate/file.ts

```diff
@@ -86,12 +86,18 @@ export function getOutputFilePath(
   assert.ok(messageName !== undefined);
 
   const outputFileName = sanitizeFileName(`${messageName}.ts`);
+  const path = [outputFileName];
+  const filePrefix = getFilePrefix(fileDescriptor);
+  const nestedMessagePrefix = getNestedMessagePrefix(parentMessageDescriptors);
 
-  return [
-    getFilePrefix(fileDescriptor),
-    getNestedMessagePrefix(parentMessageDescriptors),
-    outputFileName,
-  ].join("/");
```

Comment:
maybe we could just filter-out undefined items here?
```typescript
  return [
    getFilePrefix(fileDescriptor),
    getNestedMessagePrefix(parentMessageDescriptors),
    outputFileName,
  ].filter(part => part !== undefined && part !== '').join("/");
```

Reply:
Suggestion applied. I've confirmed it is still working as intended.